src/modules/Elsa.Workflows.Sink/Features/WorkflowSinkFeature.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using Elsa.Common.Extensions;
+using Elsa.Common.Features;
+using Elsa.Extensions;
+using Elsa.Features.Abstractions;
+using Elsa.Features.Attributes;
+using Elsa.Features.Services;
+using Elsa.MassTransit.Extensions;
+using Elsa.Mediator.Extensions;
+using Elsa.Workflows.Runtime.Notifications;
+using Elsa.Workflows.Sink.Contracts;
+using Elsa.Workflows.Sink.Implementations;
+using Elsa.Workflows.Sink.Models;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Elsa.Workflows.Sink.Features;
+
+[DependsOn(typeof(SystemClockFeature))]
+public class WorkflowSinkFeature : FeatureBase
+{
+    public WorkflowSinkFeature(IModule module) : base(module)
+    {
+    }
+
+    /// <summary>
+    /// A factory that instantiates a concrete <see cref="ISinkTransport"/>.
+    /// </summary>
+    public Func<IServiceProvider, ISinkTransport> SinkTransport { get; set; } = sp => ActivatorUtilities.CreateInstance<InProcessSinkTransport>(sp);
+    
+    /// <summary>
+    /// A factory that instantiates a concrete <see cref="IWorkflowSinkClient"/>.
+    /// </summary>
+    public Func<IServiceProvider, IWorkflowSinkClient>? WorkflowSinkClient { get; set; } = sp => ActivatorUtilities.CreateInstance<MemoryWorkflowSinkClient>(sp);
```

Comment:
As mentioned in a previous comment, I think we don't need a memory sink and can remove this one.